src/app/lib/tabs/tab-set.component.scss

```diff
@@ -0,0 +1,62 @@
+@import '../colors.scss';
+
+  .tab-bar-vertical {
+    padding-top: 15px;
+    background-color: $white;
+    width: 20%;
+    float: left;
+    height: calc(100% - 15px);
+  }
+
+  .tab-vertical {
+    background-color: $white;
+    color: $offblack;
+    padding: 15px 20px;
+    &:hover{
+      cursor: pointer;
+    }
+    &.active {
+      background-color: $slate-gray-100;
+      border-left: $blue 8px solid;
+      color: $blue;
+      font-weight: bold;
+      padding-left: 12px;
+    }
+  }
+
+  .tab-content-vertical {
+    background-color: $slate-gray-100;
+    width: calc(80% - 20px);
+    float: right;
+    height: calc(100% - 20px);
+    padding: 10px;
+  }
+
+  .tab-bar-horizontal {
+    background-color: inherit;
+    display: flex;
+    flex-direction: row;
+    border-bottom: 1px solid $slate-gray-300;
+  }
+
+  .tab-horizontal {
+    background-color: inherit;
+    color: $offblack;
+    font-size: 14px;
+    padding: 10px 20px;
+    &:hover{
+      cursor: pointer;
+    }
+    &.active {
+      border-bottom: $blue 3px solid;
```

Comment:
I should have caught this before @corytak - but can you make the horizontal bar 4px thick rather than 3?  It just looks a little thin now.  Other than that, I'm think this PR looks great.